### PR TITLE
DS-637: enable RangeSlider width change

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx [hunk 1]
```diff
@@ -213,6 +213,7 @@ export const SelectDropdownComponents = {
     );
   },
   Input: ({ id, ...props }: InputProps<IDropdownOption>) => {
+    //@ts-ignore
```
Comment (monikaracka) [inline packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx:216]: Is this necessary?

**Review by monikaracka — COMMENTED** [review]
What about w-full? Then in className, we can pass max-w-[some value]? In this way, we could avoid important in class.

File: packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx (hunk 1 above)
Comment (zuzannanap) [reply packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx:216]: yeeeessss :d

**Comment by zuzannanap** [comment]
> 

good idea :)

File: packages/design-system/src/stories/utils.tsx [hunk 2]
```diff
@@ -319,3 +319,22 @@ export const componentHeadingControl = {
   options: Object.keys(componentHeadingList),
   mapping: componentHeadingList,
 };
+
+export const rangeSliderClassNameOptions = {
+  width200: "!w-[200px]",
```
Comment (monikaracka) [inline packages/design-system/src/stories/utils.tsx:324]: You don't have to use width class with `important`. Use `max-width` class. It should work

**Review by monikaracka — APPROVED** [review]

**Pull request merged**